SocialNetwork/SocialNetwork.Core/Models/Comment.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SocialNetwork.Core.Models
+{
+    public class Comment
+    {
+        [Key]
+        public int Id { get; set; }
+        [ForeignKey("Post")]
+        public int PostId { get; set; }
```

Comment:
You should also add `public Post Post { get; set; }` navigation property and the sae way for the User.